package.json

```
@@ -1,14 +1,12 @@
 {
-  "name": "safe-token",
+  "name": "@safe-global/safe-token",
   "version": "1.0.0",
   "description": "Repository for Safe token related contracts",
   "homepage": "https://www.safe.global",
-  "license": "GPL-3.0",
+  "license": "LGPL-3.0",
   "files": [
     "contracts",
     "dist",
-    "src",
-    "test",
     "build"
```

This means that we include the build cache with the package, which I don't think we should:

```suggestion
    "build/artifacts"
```